src/controllers/controller.radar.js

```diff
@@ -102,6 +102,16 @@ module.exports = DatasetController.extend({
 		};
 	},
 
+	/**
+	 * @private
+	 */
+	_resolveElementOptions: function(element, index) {
+		if (index !== undefined) {
```

Comment:
[nit] Writing like this should save couple of bytes:
```js
return index >= 0
    ? this._resolvePointOptions(element, index)
    : this._resolveLineOptions(element);
```

Reply:
Done